### ISSUE TITLE
After updating to 6.3.2 I receive this error: Converting circular structure to JSON   starting at object with constructor 'InjectionToken'

### ISSUE THREAD

**Issue by manyx** [storybookjs/storybook#15465]
**Describe the bug**
Version 6.2.9 works fine but after I updated storybook to 6.3.2 I get this error: 
```
Converting circular structure to JSON
    --> starting at object with constructor 'InjectionToken'
    |     property 'ɵprov' -> object with constructor 'Object'
    --- property 'token' closes the circle
TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'InjectionToken'
    |     property 'ɵprov' -> object with constructor 'Object'
    --- property 'token' closes the circle
    at JSON.stringify (<anonymous>)
    at CanvasRenderer.push../node_modules/@storybook/angular/dist/ts3.9/client/preview/angular-beta/AbstractRenderer.js.AbstractRenderer.fullRendererRequired (http://localhost:6006/vendors~main.iframe.bundle.js:110980:42)
    at CanvasRenderer.<anonymous> (http://localhost:6006/vendors~main.iframe.bundle.js:110940:35)
    at step (http://localhost:6006/vendors~main.iframe.bundle.js:110872:23)
    at Object.next (http://localhost:6006/vendors~main.iframe.bundle.js:110853:53)
    at http://localhost:6006/vendors~main.iframe.bundle.js:110847:71
    at new ZoneAwarePromise (http://localhost:6006/vendors~main.iframe.bundle.js:293914:33)
    at push../node_modules/@storybook/angular/dist/ts3.9/client/preview/angular-beta/AbstractRenderer.js.__awaiter (http://localhost:6006/vendors~main.iframe.bundle.js:110843:12)
    at CanvasRenderer.push../node_modules/@storybook/angular/dist/ts3.9/client/preview/angular-beta/AbstractRenderer.js.AbstractRenderer.render 
```

**System**
Environment Info:

  System:
    OS: macOS 10.15.7
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
  Binaries:
    Node: 14.12.0 - /usr/local/bin/node
    Yarn: 1.22.5 - /usr/local/bin/yarn
    npm: 6.14.8 - /usr/local/bin/npm
  Browsers:
    Chrome: 91.0.4472.114
    Safari: 14.0
  npmPackages:
    @storybook/addon-a11y: ~6.3.2 => 6.3.2
    @storybook/addon-actions: ~6.3.2 => 6.3.2
    @storybook/addon-essentials: ~6.3.2 => 6.3.2
    @storybook/addon-links: ~6.3.2 => 6.3.2
    @storybook/angular: ~6.3.2 => 6.3.2
    @storybook/manager-webpack5: ^6.3.2 => 6.3.2
    @storybook/theming: ~6.3.2 => 6.3.2

**Additional context**
If I downgrade to 6.2 it works again.

**Comment by shilman** [comment]
Can you please create a reproduction by running `npx sb@next repro`, following the instructions, and linking it in your issue description? We prioritize issues with reproductions over those without. Thank you! 🙏

**Comment by Marklb** [comment]
I think this is the same as #15333 and should be fixed with #15410.

**Comment by shilman** [comment]
closing as dupe to #15333
